src/aleph/web/controllers/messages.py

```diff
@@ -585,3 +639,36 @@ async def view_message_status(request: web.Request):
 
     status_info = MessageStatusInfo.from_orm(message_status)
     return web.json_response(text=status_info.json())
+
+
+async def view_message_hashes(request: web.Request):
+    print("run once?")
```

Comment:
This print can be removed?

Reply:
its better to keep them separate,
here we use only "message_status" table which doesn't support all filtering like "/messages"
we should also avoid having different outputs for models and keep "/messages" relaying on "message" model